go/protocol/keybase1/extras.go

```diff
@@ -712,7 +712,9 @@ func (s *SigID) MarshalJSON() ([]byte, error) {
 
 func (f Folder) ToString() string {
```

Comment:
Sketchy looking function this is. `Folder.Private` has a big `(DEPRECATED)` in its avdl. I don't know if it would be safe to change this to use FolderType instead.

Reply:
I'll go ahead and change it and see what happens :)

Reply:
Fixed.